airflow/utils/db.py

```diff
@@ -159,7 +159,7 @@ def initdb():
     merge_conn(
         Connection(
             conn_id='http_default', conn_type='http',
-            host='https://www.google.com/'))
+            host='https://www.httpbin.org/'))
```

Comment:
I missed that on the first review. This should be mentioned in the `Updating.md` in case anyone is using `http_default` to call requests to google, don't you think?

Reply:
This is not important and will cause noise in the file.  There is a very small chance that someone would depend on it in their production environment.

Reply:
I agree with mik-laj.

Reply:
Okay 👌

Reply:
But I think we then should at least add it to the commit message. I can do that in the squash&merge process.

Reply:
Good idea. Could you please add it when you merge it? :)

Reply:
Done.

Reply:
Thank you!